cmd/flags.go

```diff
@@ -67,24 +75,64 @@ func ISPProvider() isp.Provider {
 
 	return ispProvider
 }
-
+func parseRedisURL(redisURL string) (password string, hosts []string, err error) {
+	matches := redisURLRegExp.FindStringSubmatch(redisURL)
+	if len(matches) < 3 {
+		return "", nil, fmt.Errorf("should match %s", redisURLRegExp.String())
+	}
+	return matches[1], strings.Split(matches[2], ","), nil
+}
 func RedisClient() *redis.Client {
 	if *RedisAddr == "" {
-		log.Debug("Redis not configured")
+		log.Error("Redis not configured")
```

Comment:
This is only an error if the user actually wants to use Redis. We happen to, but other consumers of ZenoDB may not.